shell.nix

```diff
@@ -58,6 +58,7 @@ let
   };
 in pkgs.mkShell {
   name = "shell";
+  LOCAL_ARCHIVE=${glibcLocales}/lib/locale/locale-archive; # works around https://github.com/tweag/ormolu/issues/38
```

Comment:
Shouldn't this be `pkgs.glibcLocales`?

Reply:
I am an absolute beginner at Nix. Should I replace "glibcLocales" with "pkgs.glibcLocales"?

Reply:
You need to wrap it in quotes and qualify `glibcLocales` with `pkgs`.